benchmark/cluster/vgg16/vgg16_tf_local.py

```diff
@@ -0,0 +1,315 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
why not use argument to decide whether to run tf as local or not?

Reply:
Done.